libnopegl/src/pass.h

```diff
@@ -30,6 +30,7 @@
 struct ngl_ctx;
 
 struct pass_params {
+    struct ngl_node *node;
```

Comment:
Is the pass allowed to change the node or can this be const?

Reply:
Made it const.

Reply:
You did?